frontend/public/components/dashboards-page/dashboards.tsx

```diff
@@ -12,9 +13,23 @@ const tabs = [
   },
 ];
 
-export const DashboardsPage: React.FC<RouteComponentProps> = ({ match }) => (
-  <React.Fragment>
-    <PageHeading title="Dashboards" detail={true} />
-    <HorizontalNav match={match} pages={tabs} noStatusBox />
-  </React.Fragment>
-);
+const _DashboardsPage: React.FC<DashboardsPageProps> = ({ match, kindsInFlight }) => {
```

Comment:
nit: Convention in console is to put the `_` at the end:

```suggestion
const DashboardsPage_: React.FC<DashboardsPageProps> = ({ match, kindsInFlight }) => {
```

Here and below.

Reply:
fixed